package.json

```diff
@@ -238,9 +236,9 @@
     "terser-webpack-plugin": "^4.1.0",
     "typescript": "4.6.4",
     "url-loader": "^4.1.0",
-    "webpack": "npm:@amoo-miki/webpack@4.46.0-rc.2",
+    "webpack": "^4.47.0",
```

Comment:
Not sure why we were using a custom branch, but is whatever we were looking for incorporated?

Reply:
My assumption here is that this was here previously to get a rc with a security fix. @AMoo-Miki is this the case or was there a patch here for some other reason?

Reply:
I was able to verify that there's no change in artifacts with this change. I diff'd the `dist` folder between `main` branch against and this branch with my changes.

I'd personally prefer to reduce these overrides to lower maintenance burden moving forward.

Reply:
The custom `webpack` release [was for Node 18+ support](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/4071#issue-1718377850) which apparently they have added with 4.47.0.